src/css/utility-classes.css

```diff
@@ -0,0 +1,13 @@
+/**
+  * Shared CSS classes. This file should remain very small, and possibly
+  * become unnecessary if we embrace a CSS-in-JS solution.
+  */
+
+.y-ellipsis {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  
+  /* overflow: hidden changes the baseline of inline-block elements, this fixes it */
+  vertical-align: bottom;
+}
```

Comment:
Are `y-text*` classes also considered utility?

Reply:
Ah sorry, that's happening in the other PR.

Reply:
merged so now showing up here :)